dkc/core/rest/user.py

```diff
@@ -19,6 +19,7 @@ class UserViewSet(GenericViewSet):
     queryset = User.objects.all()
 
     permission_classes = [AllowAny]
+    serializer_class = UserSerializer
```

Comment:
You could update line 29 too, to get the serializer from the surrounding ViewSet, but maybe you don't want to do it for actions.

Reply:
I prefer using the class explicitly.